cmd/lncli/commands.go

```diff
@@ -800,6 +805,18 @@ var closeAllChannelsCommand = cli.Command{
 			Usage: "ask for confirmation once before attempting " +
 				"to close existing channels",
 		},
+		cli.Int64Flag{
+			Name: "conf_target",
+			Usage: "(optional) the number of blocks that the " +
+				"transaction *should* confirm in, will be " +
```

Comment:
Could you rephrase this to specify that these are the closing transactions of the channels being closed?